### PR TITLE
fix: res.send(buffer)

### PR DESCRIPTION
**Pull request by khmm12**
Before it didn't send anything except headers and didn't close a connection.

### CODE REVIEW

File: packages/send/src/send.ts [hunk 1]
```diff
@@ -22,17 +22,19 @@ export const send = <Request extends Req = Req, Response extends Res = Res>(req:
...
-      // reflect this in content-type
-      const type = res.getHeader('Content-Type')
+  } else if (typeof body === 'string') {
+    // reflect this in content-type
+    const type = res.getHeader('Content-Type')
 
-      if (type && typeof type === 'string') {
-        res.setHeader('Content-Type', setCharset(type, 'utf-8'))
-      } else res.setHeader('Content-Type', setCharset('text/html', 'utf-8'))
+    if (type && typeof type === 'string') {
```
Comment (talentlessguy) [inline packages/send/src/send.ts:34]: not sure what's the point in adding line breaks?

File: packages/send/src/send.ts [hunk 2]
```diff
@@ -64,7 +66,10 @@ export const send = <Request extends Req = Req, Response extends Res = Res>(req:
       return
     } else if (Buffer.isBuffer(body)) {
       if (!res.getHeader('Content-Type')) res.setHeader('content-type', 'application/octet-stream')
-    } else encoding ? json(res)(bodyToSend, encoding) : json(res)(bodyToSend)
+      res.end(bodyToSend)
+    } else {
```
Comment (talentlessguy) [inline packages/send/src/send.ts:70]: here as well

File: packages/send/src/send.ts (hunk 1 above)
Comment (khmm12) [reply packages/send/src/send.ts:34]: ok, since it's matter of taste I reverted these lines change.

File: packages/send/src/send.ts (hunk 2 above)
Comment (khmm12) [reply packages/send/src/send.ts]: reverted as well

File: packages/send/src/send.ts (hunk 1 above)
Comment (khmm12) [reply packages/send/src/send.ts:34]: to my surprise prettier doesn't add line breaks on multiline blocks yet 🙂

**Comment by talentlessguy** [comment]
LGTM, merging

**Comment by talentlessguy** [comment]
@all-contributors add @khmm12 for bug

**Pull request merged**